### DISCUSSION TITLE
Citations / Footnotes

### DISCUSSION THREAD

**Discussion by stereo9** [outline/outline#4552] [Ideas]
Hey,

sometimes it can be useful to hint at some literature for further reading.
It would be cool to have a footnote feature that lets the editor add additional context without disrupting readability

mockup:
![image](https://user-images.githubusercontent.com/55000172/206167041-3ae288ae-1e5d-4b35-abb0-94a7a2b932a9.png)

**Comment by winkelement** [comment]
Footnotes would be nice. Most MD flavors support extended syntax afaik https://www.markdownguide.org/extended-syntax/.
